doc/lightning-setchannelfee.7.txt

```diff
@@ -0,0 +1,64 @@
+LIGHTNING-SETCHANFEE(7)                                LIGHTNING-SETCHANFEE(7)
```

Comment:
This doesn't look like asciidoc?  You should copy one of the existing .txt files and add it to the doc/Makefile, then run 'make doc-all' to generate lightning-setchannelfee.7.

Reply:
thx for the hint